core/Tracker/Action.php

```diff
@@ -206,6 +206,12 @@ protected function setActionUrl($url)
         }
     }
 
+    protected function setActionUrlWithoutExcludingParameters($url)
+    {
+        $this->rawActionUrl = PageUrl::getUrlIfLookValid($url);
+        $this->actionUrl = PageUrl::getUrlIfLookValid($url);
```

Comment:
Don't have to call the same method twice here.

Also, this skips the logging in setActionUrl. Perhaps that method can be refactored to call this new method or another one.